app/overrides/super_good/solidus_taxjar/spree/order_updater/fire_recalculated_event.rb

```diff
@@ -4,6 +4,10 @@ module SuperGood
   module SolidusTaxjar
     module Spree
       module OrderUpdater
+        # Responsible for introducing the `order_recalculated` event for older
+        # versions of Solidus (from 2.9 up until but not including 2.11) which
+        # do not have this event. TaxJar transaction reporting relies on this
+        # event firing when an order is recalculated.
         module FireRecalculatedEvent
```

Comment:
After some extensive testing with @stewart we discovered it may be time to drop this backport as the current state of the extension is no longer compatible with Solidus < 2.11 due to us referencing the persistable preferences module which was introduced in that version [here](https://github.com/SuperGoodSoft/solidus_taxjar/blob/master/app/models/super_good/solidus_taxjar/configuration.rb#L1).

Reply:
I'm on board with dropping support as we don't support Solidus 2.10 anymore!

Reply:
I'll throw a task for this on the board!